components/taxonomy/src/services.rs

```diff
@@ -158,6 +158,21 @@ pub enum ChannelKind {
     /// ```
     Ready,
 
+    /// Used when a setter takes no values.
+    ///
+    /// # JSON
+    ///
+    /// This kind is represented by string "NoKind".
+    ///
+    /// ```
+    /// use foxbox_taxonomy::services::*;
+    /// use foxbox_taxonomy::parse::*;
+    ///
+    /// let parsed = ChannelKind::from_str("\"NoKind\"").unwrap();
+    /// assert_eq!(parsed, ChannelKind::NoKind);
+    /// ```
+    NoKind,
```

Comment:
With Decentralized Taxonomy, the process should be much lighter. I have a few difficulties landing Decentralized Taxonomy, though, so this may be a few weeks before we have this lighter process.